### PR TITLE
aws-ses: add page

### PR DESCRIPTION
**Pull request by shermanhui**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
version 2010-12-01

### CODE REVIEW

File: pages/common/aws-ses.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# aws ses
+
+> CLI for AWS SES.
```
[the remark this answers is not in the document]
Comment (shermanhui) [reply pages/common/aws-ses.md:3]: Good point! I'll update this to describe what the acronym stands for, thanks for the comment.

File: pages/common/aws-ses.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`aws ses delete-receipt-rule-set --rule-set-name {{rule_set_name}} --generate-cli-skeleton`
+
+- Delete receipt rule:
+
+`aws ses delete-receipt-rule --rule-set-name {{rule_set_name}} --rule-name {{rule_name}} --generate-cli-skeleton`
+
+- Send email:
+
+`aws ses send-email --from {{from_address}} --destination "ToAddresses={{addresses}}" --message "Subject={Data={{subject_text}},Charset=utf8},Body={Text={Data={{body_text}},Charset=utf8},Html={Data={{}},Charset=utf8}}"`
```
[the remark this answers is not in the document]
Comment (shermanhui) [reply pages/common/aws-ses.md]: Oops, yes it is, I'll add the placeholder text. It should be same as the other `Data` arguments.

**Review by sbrl — APPROVED** [review]
Thanks, @shermanhui!

Can't tell if you force-pushed or whether the view changes button is broken on my end haha

**Comment by shermanhui** [comment]
@sbrl I think I might have force-pushed after the bot indicated I missed a new line at the end of the file 😅  I didn't think it warranted a separate commit merely to add a newline at the end of file 🙏🏻

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**